src/routes/missions.js

```diff
@@ -0,0 +1,54 @@
+const { Router } = require('express')
+const missionModel = require('../models/mission') 
+
+const router = Router();
+
+
+router.get('/', async (req, res) => {
+
+    let missions = await missionModel.find()
```

Comment:
Aqui voce pode usar const para evitar quem for mexer no codigo no futuro sobscrever sem querer a variavel